frontend/elements/src/entry/jig/_common/sidebar-modules/module.ts

```diff
@@ -210,6 +219,37 @@ export class _ extends LitElement {
     @property({ type: Boolean, reflect: true })
     collapsed: boolean = false;
 
+    renderDecoration() {
+        const getImage = (path: string, classes: string) =>
+            html`<img-ui class="${classes}" path="entry/jig/jiggling/${path}"></img-ui>`;
+
+        if (this.index === 0) {
+            return html`
+                ${getImage("arm-left.svg", "arm-left")}
+                ${getImage("arm-right.svg", "arm-right")}
+                ${getImage("neck-spring.svg", "neck")}
+                ${getImage("yellow/face.png", "head")}
+                ${getImage("torso-columns.svg", "torso-columns")}
+            `;
+        } else if (this.lastBottomDecoration) {
```

Comment:
This is new code. Otherwise this whole change is moving the renderDecorations into inside the element class.